src/GraphQL.Analyzers/Extensions.cs

```diff
@@ -117,6 +93,11 @@ public static Location GetMethodInvocationLocation(this MemberAccessExpressionSy
             TextSpan.FromBounds(methodNameLocation.SourceSpan.Start, argsLocation.SourceSpan.End));
     }
 
+    public static HashSet<T> ToHashSet<T>(this IEnumerable<T> source) => new(source);
+
+    public static HashSet<T> ToHashSet<T>(this IEnumerable<T> source, IEqualityComparer<T> comparer)
+        => new(source, comparer);
+
```

Comment:
.NET Standard 2.0 doesn't have a built-in `ToHashSet` extension method, only the `ToImmutableHashSet`. But `ImmutableHashSet` is significantly [slower](https://steven-giesel.com/blogPost/34e0fd95-0b3f-40f2-ba2a-36d1d4eb5601) than `HashSet`, both in terms of creation and lookup.